nix/agda-project.nix

```diff
@@ -1,16 +1,15 @@
-{ inputs, cell }:
+{ pkgs, lib, ... }:
 
-let
-  inherit (cell) library;
-  inherit (library) pkgs;
-in
-
-library.haskell-nix.hackage-project {
+pkgs.haskell-nix.hackage-project {
   name = "Agda";
 
   version = "2.6.2.2";
 
-  compiler-nix-name = library.ghc-compiler-nix-name;
+  compiler-nix-name = "ghc927";
```

Comment:
I'm unsure how much it matters what compiler version we use here... probably not that much. But at least we can do `ghc92` to get the latest GHC 9.2